README.md

```diff
@@ -1,6 +1,8 @@
 # Prisma Cloud Scan Action
-Scan container images for vulnerabilities and compliance issues using this GitHub action.
-This action is a wrapper around [twistcli](https://docs.twistlock.com/docs/compute_edition/tools/twistcli_scan_images.html) which connects to the specified Prisma Cloud Compute Console for vulnerability and compliance policy and metadata.
+The Prisma Cloud Scan Action will scan container images for vulnerabilities and compliance issues using this GitHub action. Receive immediate feedback about image vulnerabilities and compliance violations both in GitHub and in the Prisma Cloud Console. Block merges that do not meet your compliance requirements, such as updates with critical vulnerabilities.
```

Comment:
@tsmithv11 "using this GitHub action" at the end of the first sentence seems redundant to me.

Reply:
Good catch